mcs/build/Makefile

```diff
@@ -52,7 +51,6 @@ DISTFILES = \
 	corcompare.make			\
 	corcompare-api.xsl		\
 	executable.make			\
-	gensources.cs			\
```

Comment:
Looks like a typo.

Reply:
No, this file was moved to somewhere else in 8d8fd97.